Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs

```diff
@@ -43,9 +43,15 @@ private async void  WaitAndLoadLevel()
         private void OnSplashScreenClosed(IView view)
         {
             view.Closed -= OnSplashScreenClosed;
-            //_uiRoot.Open<MainMenu>();
+            _uiRoot.Open<MainMenu>();
+        }
+        
+        private void OnMainMenuClosed(IView view)
```

Comment:
This behaviour must be Main menu. Pls move this method and method WaitAndLoadLevel to main menu class

Reply:
Done